client/example_client_test.go

```diff
@@ -0,0 +1,101 @@
+package client_test
+
+import (
+	"fmt"
+
+	"github.com/go-mysql-org/go-mysql/client"
+	"github.com/go-mysql-org/go-mysql/mysql"
+)
+
+func Example() {
+	// Connect MySQL at 127.0.0.1:3306, with user root, an empty password and database test
+	conn, _ := client.Connect("127.0.0.1:3306", "root", "", "test")
```

Comment:
I think when error is not nil, we can print something to guide the developer to setup a running MySQL for this test.